Library/Formula/freeswitch.rb

```diff
@@ -146,8 +149,27 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--enable-shared",
                           "--enable-static",
-                          "--prefix=#{prefix}",
-                          "--exec_prefix=#{prefix}"
+                          "--prefix=#{etc}/freeswitch",
+                          "--exec_prefix=#{prefix}",
+                          "--with-htdocsdir=#{doc}/htdocs",
+                          "--includedir=#{include}",
+                          "--with-modinstdir=#{prefix}/mod",
+                          "--with-logfiledir=#{var}/log/#{name}",
+                          "--localstatedir=#{var}/#{name}",
+                          "--with-rundir=#{var}/run/#{name}",
+                          "--with-dbdir=#{var}/#{name}/db",
+                          "--with-soundsdir=#{var}/#{name}/sounds",
+                          "--with-recordingsdir=#{var}/#{name}/recordings",
+                          "--with-storagedir=#{var}/#{name}/storage",
+                          "--with-cachedir=#{var}/#{name}/cache"
+
+    if build.with?("flite")
+      inreplace "modules.conf", "#asr_tts/mod_flite", "asr_tts/mod_flite"
```

Comment:
I did not realise that quite a few changes have been made in `Makefile.am` for `mod_flite` from [1.4.23](https://freeswitch.org/stash/projects/FS/repos/freeswitch/browse/src/mod/asr_tts/mod_flite/Makefile.am?at=refs%2Ftags%2Fv1.4.23) to [1.6.0](https://freeswitch.org/stash/projects/FS/repos/freeswitch/browse/src/mod/asr_tts/mod_flite/Makefile.am?at=refs%2Ftags%2Fv1.6.0).